index.js

```diff
@@ -102,8 +102,10 @@ class Applesign {
     } catch (e) {
       process.exitCode = 1;
       console.error(e);
```

Comment:
if the exception is thrown theres no need to console.error it

Reply:
Agree!  Removed :D